examples/createClient.js

```diff
@@ -0,0 +1,43 @@
+// const fs = require('fs')
+require('dotenv').config()
+
+const tozStore = require('../dist/index.js')
+const Config = tozStore.Config
+const config = Config.fromObject({"version": 2,"client_id": "674badb8-53b7-483d-bf20-38b9126f7d31","client_email": "","api_url": "https://dev.e3db.com","api_key_id": "b83e58a597a0ffa7065025bcfc5cef8457bfaecd0efb1ae7368cfc595801536c","api_secret": "393826d578d397d3366301099f8b9ab9504388251d3f4971b65ac484cc73a1f2","public_key": "Efg2anzzOBUy3-6_u18vFP9VZJ-7g5NA6jcvbNiKCHk","private_key": "5vm-aVtgMgRNox2Im1bCt5ZSU-u47xK7sd02usNErhk","public_signing_key": "7mrl1r2SzkQvVAE3ANIk8uImALXO3jsYdnPWHPdGcMg","private_signing_key": "w5C0GaU_tYf5e_b0q2X1Eij2jVjQemFvNHyybGyb1MbuauXWvZLORC9UATcA0iTy4iYAtc7eOxh2c9Yc90ZwyA"})
+const Client = tozStore.init('Sodium')
+const client = new Client(config)
+
+const writeFile = async client => {
+  let writtenFile = []
+  const plainMeta = {
+    key1: 'val1',
+    key2: 'val2'
+  }
+
+  try {
+    writtenFile = await client.writeLargeFile(
+      "jane-eyre-test",
+      'examples/janeEyre.txt',
+      plainMeta
+    )
+  } catch (err) {
+    console.log(err)
+  }
+
+  console.log(writtenFile)
+}
+
+const readFile = async client => {
+  try {
+    let decryptedFile = await client.readLargeFile(
+      '3de54e05-7718-407e-af41-2a994216fdd8',
```

Comment:
Use variables that a user can set to run the example themselves with their own credentials.